src/environment.go

```diff
@@ -152,6 +152,9 @@ func (env *environment) getPlatform() string {
 }
 
 func (env *environment) runCommand(command string, args ...string) (string, error) {
+	if cmd, ok := env.commands[command]; ok {
```

Comment:
@lnu this might interest you, I removed need to put the operator in control when it comes to command caching. Something I only thought of afterwards.